### ISSUE TITLE
Cards belonging to multiple archetypes

### ISSUE THREAD

**Issue by renanlf** [AlanOC91/YGOPRODeck#370]
Hello  guys,

I am a huge fan of YGO API. It is amazing.

I was trying to help my friends to make a pure archetype tournament and want to use the API to help them see whether a card is in an archetype or not.

However, when I search for Performapal Skullcrobat Joker it responds as from Magician archetype. But I guess he is from three archetypes: Performapal (because of his name), Odd-Eyes (because of his card text), and Magician (because of his card text).

Is it possible to return an array of archetypes instead of one of them?

P.S. I am a developer (Javascript, Java, and Python). If this task needs someone to help let me know :)

**Comment by amine4567** [comment]
Even though this is a much needed feature for the api, the example you gave is bad. "Performapal Skullcrobat Joker" is only part of the Performapal archetype, he does support the Odd-Eyes and Magician archetypes but he is not part of them. 

A good example would be "Mekk-Knight Crusadia Avramax" who is part of two archetypes: Mekk-Knight and Crusadia, but the API only lists him as a Mekk-Knight.

Yugipedia is a good resource to check the archetypes to which a card belongs to, by looking in the "Search categories".
- for joker: https://yugipedia.com/wiki/Performapal_Skullcrobat_Joker
- for avramax: https://yugipedia.com/wiki/Mekk-Knight_Crusadia_Avramax

**Comment by renanlf** [comment]
Oh, I see it!

So a card belongs to an archetype only if its name mentions it? I thought card text (effects) that mention an archetype were also a condition for belonging to an archetype.

**Comment by AlanOC91** [comment]
I agree that multiple archetype definitions are needed I think the next thing I would like to implement is `supports archetype` ala Yugipedia.
